go.mod

```diff
@@ -86,6 +86,7 @@ require (
 	cloud.google.com/go/compute v1.19.3 // indirect
 	cloud.google.com/go/compute/metadata v0.2.3 // indirect
 	cloud.google.com/go/iam v1.0.1 // indirect
+	github.com/Andrew-M-C/go.jsonvalue v1.3.4 // indirect
```

Comment:
This should not be indirect – I think there's still something wrong if `go mod tidy` doesn't work from the repo root

Reply:
```
go mod why -m github.com/Andrew-M-C/go.jsonvalue
# github.com/Andrew-M-C/go.jsonvalue
github.com/rilldata/rill/runtime/server
github.com/Andrew-M-C/go.jsonvalue
```